wagtail_localize/wagtail_hooks.py

```diff
@@ -134,16 +133,9 @@ def register_submit_translation_permission():
     )
 
 
-if WAGTAIL_VERSION >= (4, 0, 0):
-
-    def set_button_icon(button, icon_name):
-        button.icon_name = icon_name
-        return button
-
-else:
-
-    def set_button_icon(button, icon_name):
-        return button
+def set_button_icon(button, icon_name):
+    button.icon_name = icon_name
+    return button
```

Comment:
another one to clean up.

e.g. https://github.com/wagtail/wagtail-localize/blob/146b4e15bfc50974dddabe424765c3ae85028768/wagtail_localize/wagtail_hooks.py#L155-L158

to become `yield wagtailadmin_widgets.Button(_("Translate this page"), url, priority=60, icon_name="wagtail-localize-language")`

(ref: https://github.com/wagtail/wagtail/blob/82d81f38b8ff889446b2d84c802ffee161f89ac9/wagtail/admin/widgets/button.py#L16)

Reply:
Deleted function `set_button_icon` and replaced all instances of `yield set_button_icon(...` to `yield wagtailadmin_widgets.Button(...`

Reply:
https://github.com/wagtail/wagtail-localize/pull/673/commits/415a156a6685ef9c2bba76e7df477fdd5bed4e99#diff-370188fc8cc675be55bcb6ce288bb8348caeee257ecab0205095fe37315e2bb7